### PR TITLE
windows agent checks test update

### PR DESCRIPTION
**Pull request by l0k0ms**
https://trello.com/c/H0OglfLH/

### CODE REVIEW

File: content/guides/agent_checks.md [hunk 1]
```diff
@@ -476,26 +476,34 @@ Troubleshooting
...
+```
+from checks import run_check
+run_check('<CHECK_NAME>')
+```
 This will output any metrics or events that the check will return.
 
+* **For agent version >= 5.12**:<br>
+Run the following script, with the proper `<CHECK_NAME>`:<br>
+`<INSTALL_DIR>/embedded/python.exe <INSTALL_DIR>agent/agent.py check <CHECK_NAME>`<br>
+For example, you can run the disk check running this command:
```
Comment (kmshultz) [inline content/guides/agent_checks.md]: For example, to run the disk check:

File: content/guides/agent_checks.md [hunk 2]
```diff
@@ -476,26 +476,34 @@ Troubleshooting
...
+```
+C:\Program Files\Datadog\Datadog Agent\embedded\python.exe C:\Program Files\Datadog\Datadog Agent\agent\agent.py check disk
+```
+**Best Practice**: cd first into the Datadog Agent folder, the command will be shorter.
```
Comment (kmshultz) [inline content/guides/agent_checks.md]: This is common sense, not necessary to say. Please omit this line.

File: content/guides/agent_checks.md [hunk 3]
```diff
@@ -476,26 +476,34 @@ Troubleshooting
...
 
-<h4>Testing Custom Checks on Windows</h4>
+#### Testing Custom Checks on Windows
 
+* **For agent version < 5.12**:<br>
 Testing custom checks on Windows is easy. The Agent install includes a file called shell.exe
 in your Program Files directory for the Datadog Agent which you can use to run python within the Agent environment.
-
-Once your check (called "my_check") is written and you have the .py and .yaml files
+Once your check (called "`<CHECK_NAME>`") is written and you have the .py and .yaml files
```
Comment (kmshultz) [inline content/guides/agent_checks.md]: (called `<CHECK_NAME>`)

(no double quotes, as <CHECK_NAME> is already in code ticks.

File: content/guides/agent_checks.md (hunk 2 above)
Comment (l0k0ms) [inline content/guides/agent_checks.md]: You would be surprised to see our support tickets :) but agreed. 👍

File: content/guides/agent_checks.md (hunk 1 above)
Comment (l0k0ms) [inline content/guides/agent_checks.md]: 👍

File: content/guides/agent_checks.md (hunk 3 above)
Comment (l0k0ms) [inline content/guides/agent_checks.md]: 👍

**Comment by l0k0ms** [comment]
@kmshultz PR updated

File: content/guides/agent_checks.md [hunk 4]
```diff
@@ -476,26 +476,33 @@ Troubleshooting
...
+    sudo -u dd-agent dd-agent check <CHECK_NAME>
 
 If your issue continues, please reach out to Support with the help page that
  lists the paths it installs.
 
-<h4>Testing Custom Checks on Windows</h4>
+#### Testing custom checks on Windows
 
+* **For agent version < 5.12**:<br>
 Testing custom checks on Windows is easy. The Agent install includes a file called shell.exe
```
Comment (kmshultz) [inline content/guides/agent_checks.md]: Let's remove "Testing custom checks on Windows is easy." Not only is it unnecessary to say, saying it in this bullet point could imply that it's NOT easy with Agent > 5.12.

Comment (l0k0ms) [inline content/guides/agent_checks.md]: 👍

**Pull request merged**